app/javascript/react/components/bands/BandShowContainer.js

```diff
@@ -36,23 +41,73 @@ const BandShowContainer = (props) => {
         throw(error)
       }
       const parsedUserObject = await response.json()
-      setUser(parsedUserObject)
+      setUser(parsedUserObject.user)
     } catch(err) {
       console.error(`Error in fetch: ${err.message}`)
     }
   }
 
   useEffect(() => {
+    fetchOneBand()
     fetchUser()
   }, [])
+
+  const addNewReview = async (formPayload) => {
+    try {
+      const response = await fetch("/api/v1/reviews", {
+        method:"POST",
+        headers:{
+          "Content-Type": "application/json",
+          "Accept": "application/json"
+        },
+        credentials:"same-origin",
+        body: JSON.stringify(formPayload)
+      })
+      if (!response.ok) {
+        const errorMessage = `${response.status} ${response.statusText}`
+        throw(new Error(errorMessage))
+      }
+      const newReview = await response.json()
+      if (newReview.review.id) {
+        setReviews([
+          ...reviews,
+          newReview.review
+        ])
+      }
+      setFormData({
+        rating: "",
+        body:"",
+        band_id:bandId
+      })
+    } catch(err) {
+      console.log(err)
+    }
+  }
+
+  const reviewTiles = reviews.map((review) => {
+    return(
+      <ReviewTiles
+        key={review.id}
+        review = {review}
+        user = {review.user}/>
```

Comment:
style is a bit bumpy here

also, remove space before and after `=` is standard react convention